core/domain/draft_upgrade_services.py

```diff
@@ -86,13 +94,8 @@ def try_upgrading_draft_to_exp_version(
         conversion_fn = getattr(DraftUpgradeUtil, conversion_fn_name)
         try:
             draft_change_list = conversion_fn(draft_change_list)
-        except Exception as error:
-            if (
-                    python_utils.UNICODE(error) ==
-                    CANNOT_CONVERT_DRAFT_EXCEPTION_MESSAGE):
-                return
-            # A blank raise will raise the last exception.
-            raise
+        except InvalidDraftConversion:
+            return
```

Comment:
Perhaps "return None", to be more explicit.

Reply:
Updated here and elsewhere